docs/en/security/encrypt.md

```diff
@@ -1,51 +1,149 @@
+- `M` represents private RSA key created by user
+- `S` represents 256-bit symmetric key generated by the JuiceFS for each file object
+- `N` represents random seed generated by the JuiceFS for each file object
+- `K` represents ciphertext obtained by `M` encrypting `S`
 
 ![Encryption At-rest](../images/encryption.png)
 
 The detailed process of data encryption is as follows:
-- Before writing to the object store, the data blocks are compressed using LZ4 or ZStandard.
+
+- Before writing to an object storage, data blocks are compressed using LZ4 or ZStandard.
 - A random 256-bit symmetric key `S` and a random seed `N` are generated for each data block.
-- AES-GCM-based encryption of each data block using `S` and `N` yields `encrypted_data`.
+- Each data block is encrypted using `S` and `N` based on AES-GCM encryption yielding `encrypted_data`.
 - To avoid the symmetric key `S` from being transmitted in clear text over the network, the symmetric key `S` is encrypted with the RSA key `M` to obtain the ciphertext `K`.
 - The encrypted data `encrypted_data`, the ciphertext `K`, and the random seed `N` are combined into an object and then written to the object storage.
 
 The steps for decrypting the data are as follows:
+
 - Read the entire encrypted object (it may be a bit larger than 4MB).
 - Parse the object data to get the ciphertext `K`, the random seed `N`, and the encrypted data `encrypted_data`.
 - Decrypt `K` with RSA key to get symmetric key `S`.
 - Decrypt the data `encrypted_data` based on AES-GCM using `S` and `N` to get the data block plaintext.
 - Decompress the data block.
 
 
-### Key Management
+### Enable Data Encryption At Rest
+
+:::note
+Data Encryption At Rest must be enabled when creating file system. The file system that has areadly enabled Data Encryption At Rest does not need to enable it again. 
+:::
+
+There are following steps to enable Data Encryption At Rest:
+
+1. Create a RSA private key
+2. Run the encrypted file system that is created by the RSA private key
+3. Mount the file system
+
+#### Step 1: create a RSA private key
+
+The RSA private key plays a crtical role in Data Encryption At Rest, usually manually generated by OpenSSL. The following command generates a 2048-bit RSA private key with a file name of `my-priv-key.pem` in the currect directory using aes256 algorithm.
+
+```shell
+openssl genrsa -out my-priv-key.pem -aes256 2048
+```
+Because of the aes256 algorithm, a passphrase (at least 4 bits) is required for this RSA private key. The passphrase can be considered as a password for encrypting the file of the RSA private key, which is also the last layer of security assurance for the RSA private key.
```

Comment:
```suggestion
Since the aes256 algorithm is in use, you will be prompted to enter a passphrase (at least 4 bits) to protect your private key. The passphrase can be considered as a password for encrypting the file of the RSA private key, which is also the last layer of security assurance for the RSA private key.
```